From 6ac5e49df667e54aef02f5181b41b233be7a4bf3 Mon Sep 17 00:00:00 2001
From: foamyguy <foamyguy@gmail.com>
Date: Mon, 16 Oct 2023 14:30:31 -0500
Subject: [PATCH] unpin sphinx and add sphinx-rtd-theme to docs reqs

---
 docs/requirements.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/docs/requirements.txt b/docs/requirements.txt
index 797aa04..979f568 100644
--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -2,5 +2,6 @@
 #
 # SPDX-License-Identifier: Unlicense
 
-sphinx>=4.0.0
+sphinx
 sphinxcontrib-jquery
+sphinx-rtd-theme
-- 
2.34.1

